### ISSUE TITLE
Emojo only toots should increase emojo size by 3-4 times

### ISSUE THREAD

**Issue by shleeable** [mastodon/mastodon#26456]
### Pitch

![image](https://github.com/mastodon/mastodon/assets/17537000/db85695f-f71a-47d8-be8c-9a4075bce4da)

### Motivation

This is just adds a little personality and human readability into the emojo reply.

**Comment by JustDalek** [comment]
I agree, I would even love emoji to be slightly bigger if its on it's own independent line tbh

**Comment by shleeable** [comment]
@ChaosExAnima @diondiondion because you're both looking into the emoji code (I think). Please consider this during your refactoring. x

note: renchap had previously looked into this and made me aware of the added complexity 
https://github.com/mastodon/mastodon/pull/26458#issuecomment-3307231502
